### PR TITLE
Fix DigitisedBy on Edit Track form and Uploader fixes.

### LINKED ISSUE #717: Editing Library also sets DigitizedBy to the editior.
LloydW:
21:34
$track->setDigitised($data['digitised']);
21:34
I bet that the fact it does that every time means it updates
21:35
Yes, yes it does
21:35
'UPDATE rec_track SET digitised=$1, digitisedby=$2 WHERE trackid=$3',
21:35
$digitised ? ['t', $_SESSION['memberid'], $this->getID()] : ['f', null, $this->getID()]
21:35
I can see the potential for improvement there

### PR DESCRIPTION
**Pull request by mstratford**
Fixes #717 
Works towards #710

### CODE REVIEW

**Comment by mstratford** [comment]
I've tested that editing tracks no longer changes the digitised by, about to test new track uploads.

**Comment by mstratford** [comment]
Seems to still work with uploading new tracks.

**Comment by mstratford** [comment]
Ready for merge now 👍

File: src/Classes/MyRadio/CoreUtils.php [hunk 1]
```diff
@@ -257,10 +257,17 @@ public static function encodeTrack($tmpfile, $dbfile)
         } elseif (!file_exists($dbfile.'.mp3') || !file_exists($dbfile.'.ogg')) {
             throw new MyRadioException('Conversion failed', 500);
         }
-
-        rename($tmpfile, "{$dbfile}.mp3.orig");
-        if (!file_exists("{$dbfile}.mp3.orig")) {
-            throw new MyRadioException('Could not move file to library.');
+        $orig_new_filename = $dbfile .".mp3.orig";
+        copy($tmpfile, $orig_new_filename);
```
Comment (LordAro) [inline src/Classes/MyRadio/CoreUtils.php]: Maybe a comment or two about why it's copy & unlink?

Comment (mstratford) [reply src/Classes/MyRadio/CoreUtils.php]: Oh, I put it in the relevant commit comment, i'll copy it over.

**Pull request merged**